### DISCUSSION TITLE
Not getting anything like the openpos stick man....

### DISCUSSION THREAD

**Discussion by Pawapluie** [Mikubill/sd-webui-controlnet#334] [Q&A] [answered]
Hey, i wanted some help, cause I tried to use Openpos model but nothing happen, I tried with many option.... playing with the weight, the guidance, the resize, etc,.... but i'm always getting something else... Someone knows why ?

<img width="293" alt="Screenshot 2023-02-24 023239" src="https://user-images.githubusercontent.com/125764425/221070180-db10df45-5678-4b0d-ad7a-402de8e2c91a.png">
<img width="964" alt="image" src="https://user-images.githubusercontent.com/125764425/221071500-90d95910-b271-4127-9290-7eb90f217481.png">

<img width="602" alt="image" src="https://user-images.githubusercontent.com/125764425/221070241-6276a326-98fb-4a88-8b57-f7434598ab6e.png">

**Comment by jmilez** [comment]
what does ControlNet show for the mask it renders (should show under the rendered image)?  I had a similar problem, the mask was always black no matter what I did.  I checked the extensions tab and both ControlNet and OpenPose Editor had updates a day after I installed them. After they were updated, it worked great.

**Comment by Pawapluie** [comment]
hey, first the mask was black then the mask was well rendered but still the picture wasn't following and now (after the update), I don't have any mask rendered after the generation. I just have the picture not even a black empty mask..... (sry for my poor eng).

**Comment by Pawapluie** [comment, marked as answer]
ok found out, i just reinstalled the whole thing ^^
